vc-verifier/kotlin/vcverifier/src/main/java/io/mosip/vercred/vcverifier/CredentialsVerifier.kt

```diff
@@ -1,175 +1,28 @@
 package io.mosip.vercred.vcverifier
 
-import android.security.KeyStoreException
 import android.util.Log
-import com.fasterxml.jackson.databind.ObjectMapper
-import com.fasterxml.jackson.databind.node.ObjectNode
-import com.nimbusds.jose.JWSObject
-import foundation.identity.jsonld.ConfigurableDocumentLoader
-import foundation.identity.jsonld.JsonLDObject
-import info.weboftrust.ldsignatures.LdProof
-import info.weboftrust.ldsignatures.canonicalizer.URDNA2015Canonicalizer
-import info.weboftrust.ldsignatures.util.JWSUtil
-import io.mosip.vercred.vcverifier.constants.CredentialVerifierConstants
-import io.mosip.vercred.vcverifier.exception.ProofDocumentNotFoundException
-import io.mosip.vercred.vcverifier.exception.ProofTypeNotSupportedException
-import io.mosip.vercred.vcverifier.exception.PublicKeyNotFoundException
-import io.mosip.vercred.vcverifier.exception.SignatureVerificationException
-import io.mosip.vercred.vcverifier.exception.UnknownException
-import io.mosip.vercred.vcverifier.utils.Util
-import okhttp3.OkHttpClient
-import okhttp3.Request
-import org.bouncycastle.util.io.pem.PemReader
-import java.io.IOException
-import java.io.StringReader
-import java.net.URI
-import java.security.KeyFactory
-import java.security.KeyManagementException
-import java.security.NoSuchAlgorithmException
-import java.security.PublicKey
-import java.security.Signature
-import java.security.cert.CertificateException
-import java.security.spec.MGF1ParameterSpec
-import java.security.spec.PSSParameterSpec
-import java.security.spec.X509EncodedKeySpec
-import java.util.Objects
+import io.mosip.vercred.vcverifier.constants.CredentialFormat
+import io.mosip.vercred.vcverifier.credentialverifier.CredentialVerifierFactory
 
 
 class CredentialsVerifier {
```

Comment:
Need inputs on of if the class name `CredentialsVerifier` needs modification, If change is required will be creating a new class which will eventually have the new method - verify

Reply:
Leave this classname as is